oauth/authentication-oauth2.0.md

```diff
@@ -9,26 +9,26 @@ SolveXia API provides two main methods for generating access tokens: **Client Cr
 
 We strongly recommend using Authorise Code Flow for client production applications. 
 The Client Credentials Flow is intended to be used by internal scripts or for testing purposes. 
-Public third party applications that rely on GitHub for authentication should not ask for or collect SolveXia user credentials.
+Public third party applications that rely on SolveXia for authentication should not ask for or collect SolveXia user credentials.
 Instead, they must use the Authorise Code Flow web flow.
```

Comment:
The majority of the documentation uses American English ("Authorization", "Authorize"). For consistency, change "Authorisation" to "Authorization" or maybe we can change the entire document to have British English.

Reply:
@hal9000-hub I've changed everything to follow American English. The reason being–avoid user confusion because "authorize" in the urls is spelled that way.